2025-03-28 15:01:14,573 - INFO - [api_client.py:20] - Intentando obtener el token...
2025-03-28 15:01:14,678 - INFO - [api_client.py:24] - Código de estado: 200
2025-03-28 15:01:14,678 - INFO - [api_client.py:25] - Respuesta cruda: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1bmlxdWVfbmFtZSI6IjgzMDEyOTAyNCIsIm5iZiI6MTc0MzE5MjA3NCwiZXhwIjoxNzQzMjc4NDc0LCJpYXQiOjE3NDMxOTIwNzQsImlzcyI6Ik1pc0ZhY3R1cmFzIn0._Ppiai71ZbHqwKTY65NQtR5c3gvErMu45se-KYpZ55w"
2025-03-28 15:01:14,679 - INFO - [api_client.py:41] - Token obtenido correctamente: eyJhb...pZ55w
2025-03-28 15:01:14,679 - INFO - [api_client.py:42] - Tiempo de obtención: 0m 0s
2025-03-28 15:01:15,726 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-28 15:01:16,582 - INFO - [document_processor.py:259] - Se encontraron 1 documentos pendientes
2025-03-28 15:01:16,582 - WARNING - [document_processor.py:273] - Tipo de documento desconocido: FV . Se usará 'FV' por defecto.
2025-03-28 15:01:16,583 - INFO - [document_processor.py:277] - Procesando DocumentID:                                                    con tipo de documento: FV 
2025-03-28 15:01:16,583 - INFO - [api_client.py:70] - Consultando detalles para DocumentID:                                                    con DocumentType: 1
2025-03-28 15:01:16,647 - ERROR - [api_client.py:81] - Error al consultar DocumentID                                                   : 405 Client Error: Method Not Allowed for url: https://www.misfacturas.com.co/integrationAPI_2/api/GetDocumentStatus?SchemaID=31&DocumentType=1&IDNumber=830129024&DocumentID=++++++++++++++++++++++++++++++++++++++++++++++++++
2025-03-28 15:01:16,908 - INFO - [document_processor.py:291] - Procesamiento de documentos pendientes completado
2025-03-28 15:01:17,988 - INFO - [database.py:62] - Conexión exitosa a la primera base de datos.
2025-03-28 15:01:22,953 - INFO - [email_notifier.py:27] - Correo enviado correctamente: Errores en la base de datos SAP
2025-03-28 15:01:23,506 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-28 15:01:28,046 - INFO - [email_notifier.py:27] - Correo enviado correctamente: Errores en la base de datos Tabla de Control
2025-03-28 15:01:28,046 - INFO - [main.py:37] - Proceso finalizado exitosamente.
2025-03-28 15:05:41,899 - INFO - [api_client.py:20] - Intentando obtener el token...
2025-03-28 15:05:42,503 - INFO - [api_client.py:24] - Código de estado: 200
2025-03-28 15:05:42,503 - INFO - [api_client.py:25] - Respuesta cruda: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1bmlxdWVfbmFtZSI6IjgzMDEyOTAyNCIsIm5iZiI6MTc0MzE5MjM0MiwiZXhwIjoxNzQzMjc4NzQyLCJpYXQiOjE3NDMxOTIzNDIsImlzcyI6Ik1pc0ZhY3R1cmFzIn0._TeLXbBF_bWZ8OMBQBQjdV3tNkDIOZ6XvmiObsHT3wU"
2025-03-28 15:05:42,503 - INFO - [api_client.py:41] - Token obtenido correctamente: eyJhb...HT3wU
2025-03-28 15:05:42,504 - INFO - [api_client.py:42] - Tiempo de obtención: 0m 0s
2025-03-28 15:05:43,682 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-28 15:05:44,515 - INFO - [document_processor.py:259] - Se encontraron 1 documentos pendientes
2025-03-28 15:05:44,515 - WARNING - [document_processor.py:268] - Se encontró un documento sin DocumentID válido. Se omitirá este documento.
2025-03-28 15:05:44,734 - INFO - [document_processor.py:287] - Procesamiento de documentos pendientes completado
2025-03-28 15:05:45,771 - INFO - [database.py:62] - Conexión exitosa a la primera base de datos.
2025-03-28 15:05:49,671 - INFO - [email_notifier.py:27] - Correo enviado correctamente: Errores en la base de datos SAP
2025-03-28 15:05:50,197 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-28 15:05:54,906 - INFO - [email_notifier.py:27] - Correo enviado correctamente: Errores en la base de datos Tabla de Control
2025-03-28 15:05:54,906 - INFO - [main.py:37] - Proceso finalizado exitosamente.
2025-03-28 15:21:31,998 - INFO - [api_client.py:20] - Intentando obtener el token...
2025-03-28 15:21:32,640 - INFO - [api_client.py:24] - Código de estado: 200
2025-03-28 15:21:32,642 - INFO - [api_client.py:25] - Respuesta cruda: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1bmlxdWVfbmFtZSI6IjgzMDEyOTAyNCIsIm5iZiI6MTc0MzE5MzI5MiwiZXhwIjoxNzQzMjc5NjkyLCJpYXQiOjE3NDMxOTMyOTIsImlzcyI6Ik1pc0ZhY3R1cmFzIn0.lKjilllpX4gWFJCMVN1N1vVjpnqo9H6B69jZI6Z2k4c"
2025-03-28 15:21:32,642 - INFO - [api_client.py:41] - Token obtenido correctamente: eyJhb...Z2k4c
2025-03-28 15:21:32,642 - INFO - [api_client.py:42] - Tiempo de obtención: 0m 0s
2025-03-28 15:21:33,777 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-28 15:21:34,957 - INFO - [document_processor.py:259] - Se encontraron 1 documentos pendientes
2025-03-28 15:21:34,958 - WARNING - [document_processor.py:272] - Tipo de documento desconocido: FV . Se usará 'FV' por defecto.
2025-03-28 15:21:34,958 - INFO - [api_client.py:70] - Consultando detalles para DocumentID:                                                    con DocumentType: 1
2025-03-28 15:21:35,094 - ERROR - [api_client.py:81] - Error al consultar DocumentID                                                   : 405 Client Error: Method Not Allowed for url: https://www.misfacturas.com.co/integrationAPI_2/api/GetDocumentStatus?SchemaID=31&DocumentType=1&IDNumber=830129024&DocumentID=++++++++++++++++++++++++++++++++++++++++++++++++++
2025-03-28 15:21:35,299 - INFO - [document_processor.py:293] - Procesamiento de documentos pendientes completado
2025-03-28 15:21:36,387 - INFO - [database.py:62] - Conexión exitosa a la primera base de datos.
2025-03-28 15:21:53,914 - INFO - [api_client.py:20] - Intentando obtener el token...
2025-03-28 15:21:54,044 - INFO - [api_client.py:24] - Código de estado: 200
2025-03-28 15:21:54,045 - INFO - [api_client.py:25] - Respuesta cruda: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1bmlxdWVfbmFtZSI6IjgzMDEyOTAyNCIsIm5iZiI6MTc0MzE5MzMxMywiZXhwIjoxNzQzMjc5NzEzLCJpYXQiOjE3NDMxOTMzMTMsImlzcyI6Ik1pc0ZhY3R1cmFzIn0.B65BRtKcDMBLEoXMRWVqnLJwRZJOgE9Afaps49W8x34"
2025-03-28 15:21:54,045 - INFO - [api_client.py:41] - Token obtenido correctamente: eyJhb...W8x34
2025-03-28 15:21:54,045 - INFO - [api_client.py:42] - Tiempo de obtención: 0m 0s
2025-03-28 15:21:55,231 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-28 15:21:56,078 - INFO - [document_processor.py:259] - Se encontraron 1 documentos pendientes
2025-03-28 15:21:56,078 - WARNING - [document_processor.py:272] - Tipo de documento desconocido: FV . Se usará 'FV' por defecto.
2025-03-28 15:21:56,078 - INFO - [api_client.py:70] - Consultando detalles para DocumentID:                                                    con DocumentType: 1
2025-03-28 15:21:56,147 - ERROR - [api_client.py:81] - Error al consultar DocumentID                                                   : 405 Client Error: Method Not Allowed for url: https://www.misfacturas.com.co/integrationAPI_2/api/GetDocumentStatus?SchemaID=31&DocumentType=1&IDNumber=830129024&DocumentID=++++++++++++++++++++++++++++++++++++++++++++++++++
2025-03-28 15:21:56,361 - INFO - [document_processor.py:293] - Procesamiento de documentos pendientes completado
2025-03-28 15:21:57,501 - INFO - [database.py:62] - Conexión exitosa a la primera base de datos.
2025-03-28 15:22:02,407 - INFO - [email_notifier.py:27] - Correo enviado correctamente: Errores en la base de datos SAP
2025-03-28 15:22:02,966 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-28 15:22:07,634 - INFO - [email_notifier.py:27] - Correo enviado correctamente: Errores en la base de datos Tabla de Control
2025-03-28 15:22:07,634 - INFO - [main.py:37] - Proceso finalizado exitosamente.
2025-03-28 15:24:28,067 - INFO - [api_client.py:20] - Intentando obtener el token...
2025-03-28 15:24:28,205 - INFO - [api_client.py:24] - Código de estado: 200
2025-03-28 15:24:28,205 - INFO - [api_client.py:25] - Respuesta cruda: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1bmlxdWVfbmFtZSI6IjgzMDEyOTAyNCIsIm5iZiI6MTc0MzE5MzQ2NywiZXhwIjoxNzQzMjc5ODY3LCJpYXQiOjE3NDMxOTM0NjcsImlzcyI6Ik1pc0ZhY3R1cmFzIn0.CVhgEYgbwT0j1IZx-VaTKhzrGrHAd0bYXZGPH1mTRgw"
2025-03-28 15:24:28,205 - INFO - [api_client.py:41] - Token obtenido correctamente: eyJhb...mTRgw
2025-03-28 15:24:28,205 - INFO - [api_client.py:42] - Tiempo de obtención: 0m 0s
2025-03-28 15:24:29,365 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-28 15:24:30,808 - INFO - [document_processor.py:259] - Se encontraron 1 documentos pendientes
2025-03-28 15:24:30,808 - WARNING - [document_processor.py:272] - Tipo de documento desconocido: FV . Se usará 'FV' por defecto.
2025-03-28 15:24:30,809 - ERROR - [document_processor.py:296] - Error general al procesar documentos pendientes: type object 'Settings' has no attribute 'API_SCHEMA_ID'
2025-03-28 15:24:32,104 - INFO - [database.py:62] - Conexión exitosa a la primera base de datos.
2025-03-28 15:24:37,124 - INFO - [email_notifier.py:27] - Correo enviado correctamente: Errores en la base de datos SAP
2025-03-28 15:24:37,691 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-28 15:24:41,110 - INFO - [email_notifier.py:27] - Correo enviado correctamente: Errores en la base de datos Tabla de Control
2025-03-28 15:24:41,111 - INFO - [main.py:37] - Proceso finalizado exitosamente.
2025-03-28 15:52:49,102 - INFO - [api_client.py:20] - Intentando obtener el token...
2025-03-28 15:52:49,511 - INFO - [api_client.py:24] - Código de estado: 200
2025-03-28 15:52:49,512 - INFO - [api_client.py:25] - Respuesta cruda: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1bmlxdWVfbmFtZSI6IjgzMDEyOTAyNCIsIm5iZiI6MTc0MzE5NTE2OSwiZXhwIjoxNzQzMjgxNTY5LCJpYXQiOjE3NDMxOTUxNjksImlzcyI6Ik1pc0ZhY3R1cmFzIn0.chqwVpkmZaf-yUq_Z8WbG06F8LjYKJ1-zCoVWokUO84"
2025-03-28 15:52:49,513 - INFO - [api_client.py:41] - Token obtenido correctamente: eyJhb...kUO84
2025-03-28 15:52:49,513 - INFO - [api_client.py:42] - Tiempo de obtención: 0m 0s
2025-03-28 15:52:50,711 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-28 15:52:51,535 - INFO - [document_processor.py:284] - Se encontraron 1 documentos pendientes
2025-03-28 15:52:51,536 - WARNING - [document_processor.py:297] - Tipo de documento desconocido: FV . Se usará 'FV' por defecto.
2025-03-28 15:52:51,536 - INFO - [api_client.py:70] - Consultando detalles para DocumentID:                                                    con DocumentType: FV
2025-03-28 15:52:51,564 - ERROR - [api_client.py:81] - Error al consultar DocumentID                                                   : 404 Client Error: Not Found for url: https://www.misfacturas.com.co/integrationAPI_2/api/GetDocumentStatus
2025-03-28 15:52:51,779 - INFO - [document_processor.py:318] - Procesamiento de documentos pendientes completado
2025-03-28 15:52:53,158 - INFO - [database.py:62] - Conexión exitosa a la primera base de datos.
2025-03-28 15:52:58,915 - INFO - [email_notifier.py:27] - Correo enviado correctamente: Errores en la base de datos SAP
2025-03-28 15:52:59,444 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-28 15:53:04,186 - INFO - [email_notifier.py:27] - Correo enviado correctamente: Errores en la base de datos Tabla de Control
2025-03-28 15:53:04,187 - INFO - [main.py:37] - Proceso finalizado exitosamente.
2025-03-28 15:53:05,544 - INFO - [api_client.py:20] - Intentando obtener el token...
2025-03-28 15:53:05,667 - INFO - [api_client.py:24] - Código de estado: 200
2025-03-28 15:53:05,668 - INFO - [api_client.py:25] - Respuesta cruda: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1bmlxdWVfbmFtZSI6IjgzMDEyOTAyNCIsIm5iZiI6MTc0MzE5NTE4NSwiZXhwIjoxNzQzMjgxNTg1LCJpYXQiOjE3NDMxOTUxODUsImlzcyI6Ik1pc0ZhY3R1cmFzIn0.QyFv7j2e6dSSDcnHEbgCqgk0C2hQCQKccS-xqCCKtB4"
2025-03-28 15:53:05,668 - INFO - [api_client.py:41] - Token obtenido correctamente: eyJhb...CKtB4
2025-03-28 15:53:05,668 - INFO - [api_client.py:42] - Tiempo de obtención: 0m 0s
2025-03-28 15:53:06,845 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-28 15:53:08,071 - INFO - [document_processor.py:284] - Se encontraron 1 documentos pendientes
2025-03-28 15:53:08,071 - WARNING - [document_processor.py:297] - Tipo de documento desconocido: FV . Se usará 'FV' por defecto.
2025-03-28 15:53:08,071 - INFO - [api_client.py:70] - Consultando detalles para DocumentID:                                                    con DocumentType: FV
2025-03-28 15:53:08,102 - ERROR - [api_client.py:81] - Error al consultar DocumentID                                                   : 404 Client Error: Not Found for url: https://www.misfacturas.com.co/integrationAPI_2/api/GetDocumentStatus
2025-03-28 15:53:08,318 - INFO - [document_processor.py:318] - Procesamiento de documentos pendientes completado
2025-03-28 15:53:09,386 - INFO - [database.py:62] - Conexión exitosa a la primera base de datos.
2025-03-28 15:53:13,516 - INFO - [email_notifier.py:27] - Correo enviado correctamente: Errores en la base de datos SAP
2025-03-28 15:53:14,041 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-28 15:53:18,712 - INFO - [email_notifier.py:27] - Correo enviado correctamente: Errores en la base de datos Tabla de Control
2025-03-28 15:53:18,712 - INFO - [main.py:37] - Proceso finalizado exitosamente.
2025-03-31 10:04:59,822 - INFO - [api_client.py:20] - Intentando obtener el token...
2025-03-31 10:05:00,259 - INFO - [api_client.py:24] - Código de estado: 200
2025-03-31 10:05:00,283 - INFO - [api_client.py:25] - Respuesta cruda: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1bmlxdWVfbmFtZSI6IjgzMDEyOTAyNCIsIm5iZiI6MTc0MzQzMzUwMCwiZXhwIjoxNzQzNTE5OTAwLCJpYXQiOjE3NDM0MzM1MDAsImlzcyI6Ik1pc0ZhY3R1cmFzIn0.M1FrK8V489OfVtYd01rQrJMfwsSEeChh-1-VNPt7tOk"
2025-03-31 10:05:00,283 - INFO - [api_client.py:41] - Token obtenido correctamente: eyJhb...t7tOk
2025-03-31 10:05:00,284 - INFO - [api_client.py:42] - Tiempo de obtención: 0m 0s
2025-03-31 10:05:01,676 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-31 10:05:02,929 - INFO - [document_processor.py:260] - Se encontraron 1 documentos pendientes
2025-03-31 10:05:02,930 - WARNING - [document_processor.py:273] - Tipo de documento desconocido: FV . Se usará 'FV' por defecto.
2025-03-31 10:05:02,930 - INFO - [api_client.py:70] - Consultando detalles para DocumentID: 445f27f2-54f4-4d80-bb1d-52b687e92dbe               con DocumentType: FV
2025-03-31 10:05:02,963 - ERROR - [api_client.py:81] - Error al consultar DocumentID 445f27f2-54f4-4d80-bb1d-52b687e92dbe              : 404 Client Error: Not Found for url: https://www.misfacturas.com.co/integrationAPI_2/api/GetDocumentStatus
2025-03-31 10:05:03,206 - INFO - [document_processor.py:294] - Procesamiento de documentos pendientes completado
2025-03-31 10:05:04,361 - INFO - [database.py:62] - Conexión exitosa a la primera base de datos.
2025-03-31 10:05:04,835 - INFO - [email_notifier.py:31] - No se encontraron errores en la base de datos para: Errores en la base de datos SAP
2025-03-31 10:05:04,944 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-31 10:05:09,607 - INFO - [email_notifier.py:27] - Correo enviado correctamente: Errores en la base de datos Tabla de Control
2025-03-31 10:05:09,607 - INFO - [main.py:37] - Proceso finalizado exitosamente.
2025-03-31 10:05:29,649 - INFO - [api_client.py:20] - Intentando obtener el token...
2025-03-31 10:05:29,727 - INFO - [api_client.py:24] - Código de estado: 200
2025-03-31 10:05:29,728 - INFO - [api_client.py:25] - Respuesta cruda: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1bmlxdWVfbmFtZSI6IjgzMDEyOTAyNCIsIm5iZiI6MTc0MzQzMzUyOSwiZXhwIjoxNzQzNTE5OTI5LCJpYXQiOjE3NDM0MzM1MjksImlzcyI6Ik1pc0ZhY3R1cmFzIn0.bWpIDUEH80kkhI3lzKkpbGrSnQDDeyinArE49qZI26g"
2025-03-31 10:05:29,729 - INFO - [api_client.py:41] - Token obtenido correctamente: eyJhb...ZI26g
2025-03-31 10:05:29,729 - INFO - [api_client.py:42] - Tiempo de obtención: 0m 0s
2025-03-31 10:05:30,906 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-31 10:05:31,770 - INFO - [document_processor.py:259] - Se encontraron 1 documentos pendientes
2025-03-31 10:05:31,771 - WARNING - [document_processor.py:272] - Tipo de documento desconocido: FV . Se usará 'FV' por defecto.
2025-03-31 10:05:31,771 - INFO - [api_client.py:70] - Consultando detalles para DocumentID: 445f27f2-54f4-4d80-bb1d-52b687e92dbe               con DocumentType: FV
2025-03-31 10:05:31,802 - ERROR - [api_client.py:81] - Error al consultar DocumentID 445f27f2-54f4-4d80-bb1d-52b687e92dbe              : 404 Client Error: Not Found for url: https://www.misfacturas.com.co/integrationAPI_2/api/GetDocumentStatus
2025-03-31 10:05:32,019 - INFO - [document_processor.py:293] - Procesamiento de documentos pendientes completado
2025-03-31 10:05:33,095 - INFO - [database.py:62] - Conexión exitosa a la primera base de datos.
2025-03-31 10:05:38,139 - INFO - [email_notifier.py:27] - Correo enviado correctamente: Errores en la base de datos SAP
2025-03-31 10:05:38,740 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-31 10:05:42,844 - INFO - [email_notifier.py:27] - Correo enviado correctamente: Errores en la base de datos Tabla de Control
2025-03-31 10:05:42,844 - INFO - [main.py:37] - Proceso finalizado exitosamente.
2025-03-31 10:09:02,643 - INFO - [api_client.py:20] - Intentando obtener el token...
2025-03-31 10:09:02,772 - INFO - [api_client.py:24] - Código de estado: 200
2025-03-31 10:09:02,772 - INFO - [api_client.py:25] - Respuesta cruda: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1bmlxdWVfbmFtZSI6IjgzMDEyOTAyNCIsIm5iZiI6MTc0MzQzMzc0MiwiZXhwIjoxNzQzNTIwMTQyLCJpYXQiOjE3NDM0MzM3NDIsImlzcyI6Ik1pc0ZhY3R1cmFzIn0.T3uy0LEcUOar3elqSHhGWU3ACN5ootnvDX4CvyzB28w"
2025-03-31 10:09:02,772 - INFO - [api_client.py:41] - Token obtenido correctamente: eyJhb...zB28w
2025-03-31 10:09:02,772 - INFO - [api_client.py:42] - Tiempo de obtención: 0m 0s
2025-03-31 10:09:04,144 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-31 10:09:05,437 - INFO - [document_processor.py:262] - Se encontraron 1 documentos pendientes
2025-03-31 10:09:05,437 - WARNING - [document_processor.py:275] - Tipo de documento desconocido: FV . Se usará 'FV' por defecto.
2025-03-31 10:09:05,437 - INFO - [api_client.py:70] - Consultando detalles para DocumentID: 445f27f2-54f4-4d80-bb1d-52b687e92dbe               con DocumentType: FV
2025-03-31 10:09:05,470 - ERROR - [api_client.py:81] - Error al consultar DocumentID 445f27f2-54f4-4d80-bb1d-52b687e92dbe              : 404 Client Error: Not Found for url: https://www.misfacturas.com.co/integrationAPI_2/api/GetDocumentStatus
2025-03-31 10:09:05,717 - INFO - [document_processor.py:296] - Procesamiento de documentos pendientes completado
2025-03-31 10:09:06,819 - INFO - [database.py:62] - Conexión exitosa a la primera base de datos.
2025-03-31 10:09:11,947 - INFO - [email_notifier.py:27] - Correo enviado correctamente: Errores en la base de datos SAP
2025-03-31 10:09:12,648 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-31 10:09:18,592 - INFO - [email_notifier.py:27] - Correo enviado correctamente: Errores en la base de datos Tabla de Control
2025-03-31 10:09:18,593 - INFO - [main.py:37] - Proceso finalizado exitosamente.
2025-03-31 10:14:03,531 - INFO - [api_client.py:20] - Intentando obtener el token...
2025-03-31 10:14:03,627 - INFO - [api_client.py:24] - Código de estado: 200
2025-03-31 10:14:03,628 - INFO - [api_client.py:25] - Respuesta cruda: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1bmlxdWVfbmFtZSI6IjgzMDEyOTAyNCIsIm5iZiI6MTc0MzQzNDA0MywiZXhwIjoxNzQzNTIwNDQzLCJpYXQiOjE3NDM0MzQwNDMsImlzcyI6Ik1pc0ZhY3R1cmFzIn0.jPOvAI0hoR1Rse2XW0cNalx7hSoxKdrr1fo_e1Jj8mw"
2025-03-31 10:14:03,628 - INFO - [api_client.py:41] - Token obtenido correctamente: eyJhb...Jj8mw
2025-03-31 10:14:03,628 - INFO - [api_client.py:42] - Tiempo de obtención: 0m 0s
2025-03-31 10:14:04,757 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-31 10:14:05,637 - INFO - [document_processor.py:260] - Se encontraron 1 documentos pendientes
2025-03-31 10:14:05,638 - WARNING - [document_processor.py:273] - Tipo de documento desconocido: FV . Se usará 'FV' por defecto.
2025-03-31 10:14:05,638 - INFO - [api_client.py:70] - Consultando detalles para DocumentID: 535e1bec-9e58-48b0-8f05-819b0d920f2b               con DocumentType: FV
2025-03-31 10:14:05,673 - ERROR - [api_client.py:81] - Error al consultar DocumentID 535e1bec-9e58-48b0-8f05-819b0d920f2b              : 404 Client Error: Not Found for url: https://www.misfacturas.com.co/integrationAPI_2/api/GetDocumentStatus
2025-03-31 10:14:05,882 - INFO - [document_processor.py:294] - Procesamiento de documentos pendientes completado
2025-03-31 10:14:06,950 - INFO - [database.py:62] - Conexión exitosa a la primera base de datos.
2025-03-31 10:14:07,381 - INFO - [email_notifier.py:36] - No se encontraron errores en la base de datos para: Errores en la base de datos SAP - 31/03/2025
2025-03-31 10:14:07,510 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-31 10:14:13,518 - INFO - [email_notifier.py:32] - Correo enviado correctamente: Errores en la base de datos Tabla de Control - 31/03/2025
2025-03-31 10:14:13,519 - INFO - [main.py:37] - Proceso finalizado exitosamente.
2025-03-31 10:24:34,886 - INFO - [api_client.py:20] - Intentando obtener el token...
2025-03-31 10:24:35,913 - INFO - [api_client.py:24] - Código de estado: 200
2025-03-31 10:24:35,913 - INFO - [api_client.py:25] - Respuesta cruda: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1bmlxdWVfbmFtZSI6IjgzMDEyOTAyNCIsIm5iZiI6MTc0MzQzNDY3NSwiZXhwIjoxNzQzNTIxMDc1LCJpYXQiOjE3NDM0MzQ2NzUsImlzcyI6Ik1pc0ZhY3R1cmFzIn0.8bSqfWfigVFVSFIE6vfh-FTMHnsdu84jAu6sJnPs0LQ"
2025-03-31 10:24:35,913 - INFO - [api_client.py:41] - Token obtenido correctamente: eyJhb...Ps0LQ
2025-03-31 10:24:35,915 - INFO - [api_client.py:42] - Tiempo de obtención: 0m 1s
2025-03-31 10:24:37,045 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-31 10:24:37,955 - INFO - [document_processor.py:260] - Se encontraron 1 documentos pendientes
2025-03-31 10:24:37,955 - WARNING - [document_processor.py:273] - Tipo de documento desconocido: FV . Se usará 'FV' por defecto.
2025-03-31 10:24:37,955 - INFO - [api_client.py:70] - Consultando detalles para DocumentID: 535e1bec-9e58-48b0-8f05-819b0d920f2b               con DocumentType: FV
2025-03-31 10:24:38,069 - ERROR - [api_client.py:81] - Error al consultar DocumentID 535e1bec-9e58-48b0-8f05-819b0d920f2b              : 404 Client Error: Not Found for url: https://www.misfacturas.com.co/integrationAPI_2/api/GetDocumentStatus
2025-03-31 10:24:38,324 - INFO - [document_processor.py:294] - Procesamiento de documentos pendientes completado
2025-03-31 10:24:39,567 - INFO - [database.py:62] - Conexión exitosa a la primera base de datos.
2025-03-31 10:24:39,912 - INFO - [email_notifier.py:36] - No se encontraron errores en la base de datos para: Errores en la base de datos SAP - 31/03/2025
2025-03-31 10:24:40,017 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-31 10:24:44,265 - INFO - [email_notifier.py:32] - Correo enviado correctamente: Errores en la base de datos Tabla de Control - 31/03/2025
2025-03-31 10:24:44,265 - INFO - [main.py:37] - Proceso finalizado exitosamente.
2025-03-31 10:32:08,800 - INFO - [api_client.py:20] - Intentando obtener el token...
2025-03-31 10:32:09,384 - INFO - [api_client.py:24] - Código de estado: 200
2025-03-31 10:32:09,385 - INFO - [api_client.py:25] - Respuesta cruda: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1bmlxdWVfbmFtZSI6IjgzMDEyOTAyNCIsIm5iZiI6MTc0MzQzNTEyOSwiZXhwIjoxNzQzNTIxNTI5LCJpYXQiOjE3NDM0MzUxMjksImlzcyI6Ik1pc0ZhY3R1cmFzIn0.8h51OCxq_fNxqt0a65FsipCN1kewr0hrV3u3B1aEtkc"
2025-03-31 10:32:09,385 - INFO - [api_client.py:41] - Token obtenido correctamente: eyJhb...aEtkc
2025-03-31 10:32:09,385 - INFO - [api_client.py:42] - Tiempo de obtención: 0m 0s
2025-03-31 10:32:10,788 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-31 10:32:11,774 - INFO - [document_processor.py:260] - Se encontraron 2 documentos pendientes
2025-03-31 10:32:11,775 - WARNING - [document_processor.py:273] - Tipo de documento desconocido: FV . Se usará 'FV' por defecto.
2025-03-31 10:32:11,775 - INFO - [api_client.py:70] - Consultando detalles para DocumentID: 535e1bec-9e58-48b0-8f05-819b0d920f2b               con DocumentType: FV
2025-03-31 10:32:11,849 - ERROR - [api_client.py:81] - Error al consultar DocumentID 535e1bec-9e58-48b0-8f05-819b0d920f2b              : 404 Client Error: Not Found for url: https://www.misfacturas.com.co/integrationAPI_2/api/GetDocumentStatus
2025-03-31 10:32:11,850 - WARNING - [document_processor.py:273] - Tipo de documento desconocido: FV . Se usará 'FV' por defecto.
2025-03-31 10:32:11,850 - INFO - [api_client.py:70] - Consultando detalles para DocumentID: 5b22d36f-b46d-4af8-9f14-ceb0d14bee33               con DocumentType: FV
2025-03-31 10:32:12,028 - ERROR - [api_client.py:81] - Error al consultar DocumentID 5b22d36f-b46d-4af8-9f14-ceb0d14bee33              : 404 Client Error: Not Found for url: https://www.misfacturas.com.co/integrationAPI_2/api/GetDocumentStatus
2025-03-31 10:32:12,388 - INFO - [document_processor.py:294] - Procesamiento de documentos pendientes completado
2025-03-31 10:32:13,673 - INFO - [database.py:62] - Conexión exitosa a la primera base de datos.
2025-03-31 10:32:14,226 - INFO - [email_notifier.py:36] - No se encontraron errores en la base de datos para: Errores en la base de datos SAP - 31/03/2025
2025-03-31 10:32:14,381 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-31 10:32:19,520 - INFO - [email_notifier.py:32] - Correo enviado correctamente: Errores en la base de datos Tabla de Control - 31/03/2025
2025-03-31 10:32:19,521 - INFO - [main.py:37] - Proceso finalizado exitosamente.
2025-03-31 10:35:13,238 - INFO - [api_client.py:20] - Intentando obtener el token...
2025-03-31 10:35:13,424 - INFO - [api_client.py:24] - Código de estado: 200
2025-03-31 10:35:13,425 - INFO - [api_client.py:25] - Respuesta cruda: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1bmlxdWVfbmFtZSI6IjgzMDEyOTAyNCIsIm5iZiI6MTc0MzQzNTMxMywiZXhwIjoxNzQzNTIxNzEzLCJpYXQiOjE3NDM0MzUzMTMsImlzcyI6Ik1pc0ZhY3R1cmFzIn0.N17U2ObzaVYNFauMaDI0__q2aVjfeP08Mq_lLU_U4K0"
2025-03-31 10:35:13,425 - INFO - [api_client.py:41] - Token obtenido correctamente: eyJhb..._U4K0
2025-03-31 10:35:13,425 - INFO - [api_client.py:42] - Tiempo de obtención: 0m 0s
2025-03-31 10:35:14,610 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-31 10:35:15,752 - INFO - [document_processor.py:260] - Se encontraron 2 documentos pendientes
2025-03-31 10:35:15,753 - WARNING - [document_processor.py:273] - Tipo de documento desconocido: FV . Se usará 'FV' por defecto.
2025-03-31 10:35:15,753 - INFO - [api_client.py:70] - Consultando detalles para DocumentID: 535e1bec-9e58-48b0-8f05-819b0d920f2b               con DocumentType: FV
2025-03-31 10:35:15,828 - ERROR - [api_client.py:81] - Error al consultar DocumentID 535e1bec-9e58-48b0-8f05-819b0d920f2b              : 404 Client Error: Not Found for url: https://www.misfacturas.com.co/integrationAPI_2/api/GetDocumentStatus
2025-03-31 10:35:15,829 - WARNING - [document_processor.py:273] - Tipo de documento desconocido: FV . Se usará 'FV' por defecto.
2025-03-31 10:35:15,829 - INFO - [api_client.py:70] - Consultando detalles para DocumentID: 5b22d36f-b46d-4af8-9f14-ceb0d14bee33               con DocumentType: FV
2025-03-31 10:35:15,936 - ERROR - [api_client.py:81] - Error al consultar DocumentID 5b22d36f-b46d-4af8-9f14-ceb0d14bee33              : 404 Client Error: Not Found for url: https://www.misfacturas.com.co/integrationAPI_2/api/GetDocumentStatus
2025-03-31 10:35:16,164 - INFO - [document_processor.py:294] - Procesamiento de documentos pendientes completado
2025-03-31 10:35:17,205 - INFO - [database.py:62] - Conexión exitosa a la primera base de datos.
2025-03-31 10:35:17,512 - INFO - [email_notifier.py:36] - No se encontraron errores en la base de datos para: Errores en la base de datos SAP - 31/03/2025
2025-03-31 10:35:17,623 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-31 10:35:22,390 - INFO - [email_notifier.py:32] - Correo enviado correctamente: Errores en la base de datos Tabla de Control - 31/03/2025
2025-03-31 10:35:22,391 - INFO - [main.py:37] - Proceso finalizado exitosamente.
2025-03-31 11:03:08,177 - INFO - [main.py:26] - Iniciando proceso de validación de documentos...
2025-03-31 11:03:08,177 - INFO - [api_client.py:20] - Intentando obtener el token...
2025-03-31 11:03:08,769 - INFO - [api_client.py:24] - Código de estado: 200
2025-03-31 11:03:08,770 - INFO - [api_client.py:25] - Respuesta cruda: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1bmlxdWVfbmFtZSI6IjgzMDEyOTAyNCIsIm5iZiI6MTc0MzQzNjk4OCwiZXhwIjoxNzQzNTIzMzg4LCJpYXQiOjE3NDM0MzY5ODgsImlzcyI6Ik1pc0ZhY3R1cmFzIn0.FLal3F2IxIN-EUMcJ7ws9RDFk-8vlhUCqSL5rVT5FA0"
2025-03-31 11:03:08,770 - INFO - [api_client.py:41] - Token obtenido correctamente: eyJhb...T5FA0
2025-03-31 11:03:08,770 - INFO - [api_client.py:42] - Tiempo de obtención: 0m 0s
2025-03-31 11:03:10,435 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-31 11:03:11,391 - INFO - [document_processor.py:260] - Se encontraron 5 documentos pendientes
2025-03-31 11:03:11,392 - ERROR - [document_processor.py:297] - Error general al procesar documentos pendientes: cannot access local variable 'tipo_documento' where it is not associated with a value
2025-03-31 11:03:12,859 - INFO - [database.py:62] - Conexión exitosa a la primera base de datos.
2025-03-31 11:03:17,723 - INFO - [email_notifier.py:32] - Correo enviado correctamente: Errores en la base de datos SAP - 31/03/2025
2025-03-31 11:03:18,331 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-31 11:03:23,438 - INFO - [email_notifier.py:32] - Correo enviado correctamente: Errores en la base de datos Tabla de Control - 31/03/2025
2025-03-31 11:03:23,439 - INFO - [main.py:38] - Proceso finalizado exitosamente.
2025-03-31 11:03:23,439 - INFO - [main.py:48] - Esperando 30 minutos para la siguiente ejecución...
2025-03-31 11:20:00,577 - INFO - [main.py:26] - Iniciando proceso de validación de documentos...
2025-03-31 11:20:00,578 - INFO - [api_client.py:20] - Intentando obtener el token...
2025-03-31 11:20:01,046 - INFO - [api_client.py:24] - Código de estado: 200
2025-03-31 11:20:01,046 - INFO - [api_client.py:25] - Respuesta cruda: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1bmlxdWVfbmFtZSI6IjgzMDEyOTAyNCIsIm5iZiI6MTc0MzQzODAwMCwiZXhwIjoxNzQzNTI0NDAwLCJpYXQiOjE3NDM0MzgwMDAsImlzcyI6Ik1pc0ZhY3R1cmFzIn0.lrMSmojlVgP92sKwBT23Cl7ttxTS0hAHdDbAdnLtcZE"
2025-03-31 11:20:01,048 - INFO - [api_client.py:41] - Token obtenido correctamente: eyJhb...LtcZE
2025-03-31 11:20:01,048 - INFO - [api_client.py:42] - Tiempo de obtención: 0m 0s
2025-03-31 11:20:02,230 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-31 11:20:03,088 - INFO - [document_processor.py:260] - Se encontraron 9 documentos pendientes
2025-03-31 11:20:03,089 - ERROR - [document_processor.py:297] - Error general al procesar documentos pendientes: cannot access local variable 'tipo_documento' where it is not associated with a value
2025-03-31 11:20:04,460 - INFO - [database.py:62] - Conexión exitosa a la primera base de datos.
2025-03-31 11:20:09,345 - INFO - [email_notifier.py:32] - Correo enviado correctamente: Errores en la base de datos SAP - 31/03/2025
2025-03-31 11:20:09,867 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-31 11:20:15,181 - INFO - [email_notifier.py:32] - Correo enviado correctamente: Errores en la base de datos Tabla de Control - 31/03/2025
2025-03-31 11:20:15,182 - INFO - [main.py:38] - Proceso finalizado exitosamente.
2025-03-31 11:20:15,182 - INFO - [main.py:48] - Esperando 30 minutos para la siguiente ejecución...
2025-03-31 11:22:17,946 - INFO - [main.py:26] - Iniciando proceso de validación de documentos...
2025-03-31 11:22:17,947 - INFO - [api_client.py:20] - Intentando obtener el token...
2025-03-31 11:22:18,067 - INFO - [api_client.py:24] - Código de estado: 200
2025-03-31 11:22:18,067 - INFO - [api_client.py:25] - Respuesta cruda: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1bmlxdWVfbmFtZSI6IjgzMDEyOTAyNCIsIm5iZiI6MTc0MzQzODEzNywiZXhwIjoxNzQzNTI0NTM3LCJpYXQiOjE3NDM0MzgxMzcsImlzcyI6Ik1pc0ZhY3R1cmFzIn0.mwjXu7DCXiUqzjXgohS17U8SzvI4eU6xh6zhJ-1b4Kw"
2025-03-31 11:22:18,067 - INFO - [api_client.py:41] - Token obtenido correctamente: eyJhb...1b4Kw
2025-03-31 11:22:18,067 - INFO - [api_client.py:42] - Tiempo de obtención: 0m 0s
2025-03-31 11:22:19,229 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-31 11:22:20,099 - INFO - [document_processor.py:260] - Se encontraron 9 documentos pendientes
2025-03-31 11:22:20,099 - ERROR - [document_processor.py:297] - Error general al procesar documentos pendientes: cannot access local variable 'tipo_documento' where it is not associated with a value
2025-03-31 11:22:21,432 - INFO - [database.py:62] - Conexión exitosa a la primera base de datos.
2025-03-31 11:22:26,983 - INFO - [email_notifier.py:32] - Correo enviado correctamente: Errores en la base de datos SAP - 31/03/2025
2025-03-31 11:22:27,527 - INFO - [database.py:88] - Conexión exitosa a la segunda base de datos.
2025-03-31 11:22:31,334 - INFO - [email_notifier.py:32] - Correo enviado correctamente: Errores en la base de datos Tabla de Control - 31/03/2025
2025-03-31 11:22:31,335 - INFO - [main.py:38] - Proceso finalizado exitosamente.
2025-03-31 11:22:31,335 - INFO - [main.py:48] - Esperando 30 minutos para la siguiente ejecución...
